spring-boot/pom.xml

```diff
@@ -58,15 +58,15 @@
 		<dependency>
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<version>42.2.23</version>
+			<version>42.5.0</version>
 		</dependency>
 		
 		<!-- GCLOUD Dependencies -->
 		<!-- Using CloudSqlJdbcInfoProvider bean to initialize DB connection with spring.cloud.gcp.sql.* values from application.properties -->
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-starter-sql-postgresql</artifactId>
-			<version>1.2.4.RELEASE</version>
+			<version>1.2.8.RELEASE</version>
```

Comment:
did this change on its own when you updated sentry-spring-boot and sentry-logback?

or was this part of an Upgrade Pathway per the sentry sdk instructions?

Try deploying to Staging since this affects how AppEngine->Springboot connects to CloudSQL->PostgreSQL
@Angelodaniel